docs/certs.md

```diff
@@ -0,0 +1,10 @@
+![Certification management diagram](./assets/certs.svg)
+
+## Certification management
+
+BTP Manager maintain two secrets `ca-server-cert` and `webhook-server-cert` which are used to allow communication within BTP Operator webhooks, and in result allow to create resources like ServiceInstances and ServiceBindings
+During provisioning, first there is created `ca-server-cert` which is self sign CA certificate, then based on that, application create signed cert `webhook-server-cert`, which is mounted under the deployment.
+Webhooks have CABundle field which is set to the content of ca-server-cert and this field is managed by BTP Manager.
```

Comment:
You use `caBundle`, `CA bundle`, `ca bundle` and `CABundle`. If they're not three different things, why don't you use the same spelling in the whole doc?